keel-sql/src/main/resources/db/changelog/20200511-misc-platform-review-changes.yml

```diff
@@ -0,0 +1,45 @@
+databaseChangeLog:
+- changeSet:
+    id: add-missing-primary-keys
+    author: fletch
+    changes:
+    - dropUniqueConstraint:
+        tableName: artifact_last_checked
+        constraintName: artifact_uid
+        uniqueColumns: artifact_uid
+    - dropUniqueConstraint:
+        tableName: delivery_config_last_checked
+        constraintName: delivery_config_uid
+        uniqueColumns: delivery_config_uid
+    - dropUniqueConstraint:
+        tableName: resource_last_checked
+        constraintName: resource_uid
+        uniqueColumns: resource_uid
+    - addPrimaryKey:
```

Comment:
i assume this was generated by jooq? Will it just convert these multiple primary keys to a single composite primary key?

Reply:
I am not sure, but I think the syntax here is wrong:
https://www.liquibase.org/documentation/changes/add_primary_key.html

i think it's supposed to be something like this:
```yaml
    - addPrimaryKey:
        tableName: delivery_config_last_checked
        columnNames: artifact_last_checked, delivery_config_uid, resource_last_checked
```
(instead of 3 adds)
also, assuming the order these columns is what you want - since they will comprise the PK index and will serve as the main lookup.

Reply:
Sorry, ignore me, i am an idiot... these are different tables... 🤦